python/S3_OBJECT_LOCK_ENABLED/S3_OBJECT_LOCK_ENABLED.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2017-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+'''
+#####################################
+##           Gherkin               ##
+#####################################
+Rule Name:
+  S3_OBJECT_LOCK_ENABLED
+
+Description:
+  Check whether S3 Buckets are Object Lock enabled. There should be a retention mode and retention
+  period to be applied to the bucket lock.
+
+Rationale:
+  Object Lock enabled on all S3 buckets to block object version deletion.
+
+Indicative Severity:
+  Medium
+
+Trigger:
+  Periodic
+
+Reports on:
+  AWS::S3::Bucket
+
+Rule Parameters:
+  Mode
+    The Object Lock retention mode you want to apply to new objects placed in the specified bucket
+  Days
+    The number of days that you want to specify for the default retention period.
```

Comment:
same

Reply:
The description should be about the fact that it is the minimum value expected. I'd rename it `MinimumRetentionDays`.